examples/sveltekit-pwa/pwa.js

```diff
@@ -11,16 +12,18 @@ process.env.RELOAD_SW = `${args['RELOAD_SW'] === 'true'}`
 process.env.SW = `${args['SW'] === 'true'}`
 
 const webmanifestDestinations = [
-	'./.svelte-kit/output/client/',
-	'./build/',
+	'./.svelte-kit/output/client/_app',
+	'./build/_app',
 ]
 
 const swDestinations = [
-	'./build/',
+	'./.svelte-kit/output/client',
 ]
```

Comment:
I sent https://github.com/sveltejs/kit/pull/5416 which hopefully removes the need for a lot of this